corehq/apps/app_manager/suite_xml/case_tile_templates/bha_referrals.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "bha_referrals",
+    "filename": "bha_referrals.xml",
+    "has_map": true,
+    "fields": ["Profile Information", "Age", "Gender", "ASAM Level",
```

Comment:
nit: For reusability it would be nice to see more generic identifiers like `top_left` etc.